relayer/src/services/relayer/optimism.ts

```diff
@@ -157,29 +185,12 @@ export class OptimismRelayerService extends BaseRelayerService {
     return true;
   }
 
-  async prepare(): Promise<boolean> {
-    return true;
-  }
-
-  async relayTxToL1(l2TransactionHash: TransactionHash): Promise<TransactionHash | undefined> {
+  private async relayTxToL1(l2TransactionHash: TransactionHash): Promise<TransactionHash> {
     console.log("Optimism outbox execution.");
-
-    await this.l2RpcProvider.waitForTransaction(l2TransactionHash, 1);
```

Comment:
I think we can keep this line just in case the `relayTxToL1` step is being called without a prior call to `isRelayCompleted`.

Reply:
:heavy_check_mark: